nomos-da/kzgrs-backend/src/dispersal.rs

```diff
@@ -250,6 +250,7 @@ mod tests {
         verifiers: &[DaVerifier],
     ) -> Vec<Attestation> {
         let mut attestations = Vec::new();
+        let domain_size = encoded_data.extended_data.0[0].len();
```

Comment:
I assumed this gives us the row size (for extended data)

Reply:
Yes, just for the test.